src/solc_0.8/polygon/LiquidityMining/PolygonLandWeightedSANDRewardPool.sol

```diff
@@ -162,7 +162,12 @@ contract PolygonLandWeightedSANDRewardPool is PolygonLPTokenWrapper, IRewardDist
         _contributions[account] = contribution;
     }
 
-    function computeMultiplier(address account) public onlyRewardDistributionOrAccount(account) updateReward(account) {
+    function computeMultiplier(address account)
+        public
+        nonReentrant
```

Comment:
why nonreeentrant? is there any way that you can reenter this function?

Reply:
indeed ,fixed